packages/compass-indexes/src/modules/regular-indexes.ts

```diff
@@ -508,7 +559,15 @@ export function createRegularIndex(
       await createFn(ns, spec, options);
       dispatch(indexCreationSucceeded(inProgressIndexId));
       track('Index Created', trackEvent, connectionInfoRef.current);
-
+      // See action description for details
+      if (isRollingIndexBuild) {
+        setTimeout(() => {
```

Comment:
Wondering if it is worth fussing over the timeout and clearing it. Probably not. Super rare to be adding rolling indexes and then there's only one and if the in-progress index disappeared in the meantime then it will just do nothing.

Reply:
That was my reasoning for not doing it, but I probably should put this in a comment, it's a rarely used feature and this actino is harmless when it fires, so adding more code to store the reference seemed like an overkill